### DISCUSSION TITLE
[Master-server][Schedule][V3.1.5] Schedules dont work

### DISCUSSION THREAD

**Discussion by IanLianet** [apache/dolphinscheduler#14424] [Q&A]
No new process instance was generated which is of scheduled process definition. the schedule's release state is ONLINE (which is '1' in the field 'release_state' of  t_ds_schedules table).
the log of 2 master server nodes show these:
1.
`This scheduler instance (dpd-dev-751688023101104) is still active but was recovered by another instance in the cluster.  This may cause inconsistent behavior.
`
2. 
```
[INFO] 2023-06-29 15:54:52.982 +0800 org.springframework.scheduling.quartz.LocalDataSourceJobStore:[3644] - [WorkflowInstance-0][TaskInstance-0] - ClusterManager: detected 1 failed or restarted instances.
[INFO] 2023-06-29 15:54:52.982 +0800 org.springframework.scheduling.quartz.LocalDataSourceJobStore:[3503] - [WorkflowInstance-0][TaskInstance-0] - ClusterManager: Scanning for instance "dpd-dev-751688023101104"'s failed in-progress jobs.
```

**Comment by ruanwenjun** [comment]
Have you restarted the cluster?
